corehq/apps/api/resources/v0_4.py

```diff
@@ -371,7 +375,7 @@ class Meta(CustomResourceMeta):
 class BaseApplicationResource(CouchResourceMixin, HqBaseResource, DomainSpecificResourceMixin):
 
     def obj_get_list(self, bundle, domain, **kwargs):
-        return get_apps_in_domain(domain, include_remote=False)
+        return sorted(get_apps_in_domain(domain, include_remote=False), key=attrgetter("date_created"))
```

Comment:
Do `date_created` values always have the same type? This will error if, for example, most of the time it's a `datetime`, but sometimes its `None` or `str`.

Reply:
it's always a date